extensions/wikia/XBoxGamerCardTag/XBoxGamerCardTag.php

```diff
@@ -57,7 +57,7 @@ function wfMakeXboxCard( $contents, $attributes, $parser ) {
 	else
 	{
 		$xbox_id_e = rawurlencode($xbox_id); #we use raw version because microsoft no longer accepts spaces as +, MUST be %20
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[project-id]: wikia:app